gateway-js/src/config.ts

```diff
@@ -121,69 +126,174 @@ interface GatewayConfigBase {
   experimental_didResolveQueryPlan?: Experimental_DidResolveQueryPlanCallback;
   experimental_didFailComposition?: Experimental_DidFailCompositionCallback;
   experimental_didUpdateComposition?: Experimental_DidUpdateCompositionCallback;
+  /**
+   * @deprecated use `pollIntervalInMs` instead
+   */
   experimental_pollInterval?: number;
+  pollIntervalInMs?: number;
```

Comment:
just to confirm I understand: this is becoming a real thing because it actually affects the uplink polling (in addition to affecting deprecated stuff)?

Reply:
This could be done in a separate PR if that's what you're asking, it doesn't need to be done now for everything to work.

Reply:
sorry, i was just double-checking that the reason this was becoming non-experimental rather than removed is because it is a useful feature that helps managed federation too and not just the legacy options.  and i'm pretty sure that's true.

Reply:
Yep! Your question actually made sense, I think I just read it wrong.